engine/runtime-integration-tests/src/test/java/org/enso/interpreter/runtime/ResourceManagerTest.java

```diff
@@ -22,13 +20,8 @@ public void runFinalizersAtTheEnd() {
     var obj = new ResourceToGc();
     var fn = new FnCallback();
 
-    try (var ctx = ContextUtils.createDefaultContext()) {
-      ctx.enter();
-      var ensoContext =
-          (EnsoContext)
-              ctx.getBindings(LanguageInfo.ID)
-                  .invokeMember(MethodNames.TopScope.LEAK_CONTEXT)
-                  .asHostObject();
+    try (var ctx = ContextRule.createDefault()) {
+      var ensoContext = ctx.leakContext();
```

Comment:
Better to rename `leakContext` to `ensoContext` in my opinion.

Reply:
Renamed in b28a8bee870700ccad72283ad84cd4df9fdba220